Mvc.NoFun.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22522.0
+VisualStudioVersion = 14.0.22526.0
```

Comment:
@davidfowl to avoid mixing random changes to otherwise-unaffected files into real commits